lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -114,16 +114,6 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
     }
     subscriptions.clear();
     lastEditingState = null;
-
-    // If the text element still has focus, remove focus from the editable
-    // element to cause the on-screen keyboard, if any, to hide (e.g. on iOS,
-    // Android).
-    // Otherwise, the keyboard stays on screen even when the user navigates to
-    // a different screen (e.g. by hitting the "back" button).
-    // Keep this consistent with how DefaultTextEditingStrategy does it. As of
-    // right now, the only difference is that semantic text fields do not
-    // participate in form autofill.
-    DefaultTextEditingStrategy.scheduleFocusFlutterView(activeDomElement, activeDomElementView);
```

Comment:
`TextEditingStrategy.disable` is used by `TextInput.clearClient`, `TextInput.hide`, and in one case by `TextInput.setClient`. The focus move (which previously was just `domElement.blur()`) seems to have the intention to close the virtual keyboard when the framework demands it. If we do not do anything here, does it mean that there are situations when the framework would detach and move onto the next widget, but the browser will stay focused on the text field and virtual keyboard showing? I remember we had bugs where the virtual keyboard would show up, but typing into it would not enter any text, because the `<input>` is not connected to the framework.

Reply:
That's a great point Yegor! 

If I understand correctly, the `deactivated` call suggests either app transitioned "flutter" focus to another widget, shifting the "browser" focus, or the user interacted outside the app, causing it to lose focus.